java/spacewalk-java.spec

```diff
@@ -53,97 +65,134 @@ BuildRoot:      %{_tmppath}/%{name}-%{version}-build
 BuildArch:      noarch
 ExcludeArch:    ia64 aarch64
 
-Requires:       bcel
-Requires:       c3p0 >= 0.9.1
-Requires:       classpathx-mail
+BuildRequires:  ant
+BuildRequires:  ant-apache-regexp
+BuildRequires:  ant-contrib
+BuildRequires:  ant-junit
+BuildRequires:  ant-nodeps
+BuildRequires:  antlr >= 2.7.6
+BuildRequires:  apache-commons-cli
+BuildRequires:  apache-commons-codec
+BuildRequires:  apache-commons-collections
+BuildRequires:  apache-commons-io
+#??? BuildRequires:  apache-commons-lang
```

Comment:
Nitpick : Probably should remove it, no?

Reply:
right. I commented it out to test if it is building without. It does, so I can remove it